src/lib/containers/RssFeedCards.tsx

```diff
@@ -103,12 +102,20 @@ export default class RssFeedCards extends React.Component<RssFeedCardsProps, Rss
                 >
                   <div>
                     <div className="RssFeedItemCategories">
-                    {item['categories'].map((categoryName: any, index: any) => {
-                      if (index >= MAX_CATEGORIES)
-                        return <></>
-                      // else
-                      return <div key={`${item.guid}_${categoryName}`} className="RssFeedItemCategory">{categoryName}</div>
-                    })}
+                      {item['categories'].map((categoryName: any,) => {
+                        // are we allowed to show this category/tag?
+                        const categoryNameLowerCase = categoryName.toLowerCase()
```

Comment:
it looks like categoryName could be of type string instead of any?

Reply:
nice catch, it certainly is.  updated.